crates/red_knot_python_semantic/resources/mdtest/scopes/moduletype_attrs.md

```diff
@@ -58,11 +58,10 @@ reveal_type(typing.__name__)  # revealed: str
 reveal_type(typing.__init__)  # revealed: Literal[__init__]
 
 # These come from `builtins.object`, not `types.ModuleType`:
-# TODO: we don't currently understand `types.ModuleType` as inheriting from `object`;
-# these should not reveal `Unknown`:
-reveal_type(typing.__eq__)  # revealed: Unknown
-reveal_type(typing.__class__)  # revealed: Unknown
-reveal_type(typing.__module__)  # revealed: Unknown
```

Comment:
Instances of `ModuleType` don't actually have `__module__` attributes in "real life" 😆 So it seemed like a bad thing to assert in a test, in retrospect.

We infer that they _do_ have `__module__` attributes because of that slightly-irritating inaccuracy that we discussed the other day.